### PR TITLE
Extended setup script

### PR DESCRIPTION
**Pull request by someth2say**

### CODE REVIEW

File: setup/setup.sh [hunk 1]
```diff
@@ -1,62 +1,100 @@
...
+    echo "Deleting Kubectl context ${MINIKUBE_CONTEXT} ..."
+    if ! kubectl config delete-context $MINIKUBE_CONTEXT  > /dev/null 2>&1; then
+        echo "Error while deleting config context"
         exit
     fi
+}
 
-    if ! kubectl apply -f $SCRIPTPATH/files/role-binding.yml &> /dev/null ; then
+apply_role_resources() {
+    if ! sed "s/{username}/${USERNAME}/g; s/{namespace}/${NAMESPACE_DEV}/g" $SCRIPTPATH/files/role-binding.yml | kubectl apply -f -  > /dev/null 2>&1 ; then
```
Comment (enolredhat) [inline setup/setup.sh:85]: We need to use the parameter of the function
```suggestion
    if ! sed "s/{username}/${USERNAME}/g; s/{namespace}/${1}/g" $SCRIPTPATH/files/role-binding.yml | kubectl apply -f -  > /dev/null 2>&1 ; then
```

File: setup/setup.sh [hunk 2]
```diff
@@ -1,62 +1,100 @@
-USERNAME="student"
+#! /bin/sh
+
+USERNAME=$(whoami)
```
Comment (enolredhat) [inline setup/setup.sh:3]: I don't think the system username will always match ROL's username, but I guess it's ok for the moment

**Review by enolredhat — CHANGES_REQUESTED** [review]

File: setup/setup.sh (hunk 2 above)
Comment (someth2say) [reply setup/setup.sh:3]: We can't think in terms of ROL.
There will be no ROL, just the local env (minikube) and the remote env (Dev Sandbox).

File: setup/setup.sh (hunk 1 above)
Comment (someth2say) [reply setup/setup.sh:85]: Good catch!

File: setup/setup.sh (hunk 2 above)
Comment (enolredhat) [reply setup/setup.sh:3]: True, I was thinking about the Developer Sandbox, not sure if it's the same login than ROL?

I mean, the problem we will have is how to know what is the Developer Sandbox username to use it also in Minikube. But I think there's no right answer

**Pull request merged**